probfit/pdf.pyx

```diff
@@ -514,3 +514,69 @@ cpdef double cauchy(double x, double m, double gamma):
     """
     cdef double xmg = (x-m)/gamma
     return 1/(pi*gamma*(1+xmg*xmg))
+
+
+cdef class Exponential:
+    """
+    Exponential [1]_.
+
+    .. math::
+        f(x;\\tau) =
+        \\begin{cases}
```

Comment:
What's this `cases` env for?

Reply:
To like for the cruijf distribution
<img width="581" alt="capture d ecran 2018-11-14 a 11 40 18" src="https://user-images.githubusercontent.com/24250309/48477520-1aae2880-e802-11e8-8f07-cd7641971a8d.png">

Reply:
<img width="258" alt="capture d ecran 2018-11-14 a 11 41 36" src="https://user-images.githubusercontent.com/24250309/48477575-40d3c880-e802-11e8-9b51-d910207379d1.png">

Reply:
I thought it would render OK without it. But I did not try it out … I believe you :-).